plutarch-test/src/Plutarch/Test/Golden.hs

```diff
@@ -168,17 +170,21 @@ currentGoldenKey = do
         Just path ->
           pure $ sconcat $ fmap GoldenKey path
 
-{- | Like `evaluateScript` but doesn't fail. Also returns `Script`.
+{- | Like `evalScript` but doesn't fail, and returns `Benchmark`.
```

Comment:
Comment seems outdated.

Reply:
It is accurate?

Reply:
`evalScript` doesn't fail either

Reply:
By "faIl" I meant it returning `Left EvalError`; I've clarified the comment.